wdn/templates_5.3/scss/elements/_elements.anchors.scss

```diff
@@ -4,7 +4,8 @@
 
 
 // Finds anchor tags with hrefs not containing unl.edu, and not starting with # or /
-.unl a:not([href*='unl.edu']):not([href^='#']):not([href^='/']):not(:has(img)):not(:has(svg))::after {
+.unl a[href^='http://']:not([href*='unl.edu']):not(:has(img)):not(:has(svg))::after,
+.unl a[href^='https://']:not([href*='unl.edu']):not(:has(img)):not(:has(svg))::after {
```

Comment:
Does not catch links like `<a href="//www.google.com">google.com</a>` which some how work

Need to add `a[href^="//"]...`